modules/ingress-operator.adoc

```diff
@@ -20,9 +20,12 @@ ifdef::cluster-caps[= Ingress Capability]
 == Purpose
 
 ifdef::cluster-caps[]
+The Ingress Operator provides the features for the `Ingress` capability. Ingress capability is enabled by default.
 
-The Ingress Operator provides the features for the `Ingress` capability.
-
+[IMPORTANT]
+====
+If you set `baselineCapabilitySet` to `None`, you must explicitly enable Ingress capability, because installation of a cluster fails if Ingress capability is disabled.
```

Comment:
```suggestion
If you set the `baselineCapabilitySet` field to `None`, you must explicitly enable the Ingress capability because the installation of a cluster fails if the Ingress capability is disabled.
```

Reply:
:heavy_check_mark: